src/scss/modifiers/_custom.scss

```diff
@@ -97,29 +97,32 @@
 /// @param {String} $input Type of input to customise, one of 'anchor' or 'radio'
 /// @param {Boolean} $icons A list of icons to render
```

Comment:
```suggestion
/// @param {List} $icons A list of icons to render
```

Reply:
Hmm, we should look at making oat check jsdoc types and ensure they are correct

Reply:
`oat` will be ALL POWERFUL

Reply:
https://github.com/Financial-Times/origami-authoring-tools/issues/11